src/MarshalsPsr7RequestsAndResponses.php

```diff
@@ -58,7 +58,7 @@ protected function toPsr7Response(Response $response): ResponseInterface
      *
      * This instance can turn a PSR-7 request into an HttpFoundation request.
      *
-     * @return \Symfony\Bridge\PsrHttpMessage\Factory\HttpFoundationFactory
+     * @return \Symfony\Bridge\PsrHttpMessage\Factory\HttpFoundationFactoryInterface
```

Comment:
Maybe is `Symfony\Bridge\PsrHttpMessage\HttpFoundationFactoryInterface`?

Reply:
It is already changed to ```HttpFoundationFactoryInterface```.

Reply:
But the FQN is not correct. Check this: https://github.com/symfony/psr-http-message-bridge/blob/main/HttpFoundationFactoryInterface.php.

Reply:
I checked and it seems correct as the last change.

Reply:
No, it is not correct. The `HttpFoundationFactoryInterface` is imported as `Symfony\Bridge\PsrHttpMessage\HttpFoundationFactoryInterface` in the same file on line 14:

https://github.com/laravel/octane/blob/dc393ebf101c880d4b96c5560d2d50bea6adb91a/src/MarshalsPsr7RequestsAndResponses.php#L14

Reply:
Changed, yes you're right.
thanks!